views/partials/header/fxa-menu.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="fxa-menu-wrapper">
-  <div id="avatar-wrapper" class="avatar-wrapper {{ addClass }}" tabindex="0" aria-label="{{ getString "open-fxa-menu" }}">
+  <div id="avatar-wrapper" data-ga-link="" data-event-category="avatar-menu" data-event-label="open-fxa-menu-open" class="avatar-wrapper {{ addClass }}" tabindex="0" aria-label="{{ getString "open-fxa-menu" }}">
```

Comment:
Adding GA event tracking for anytime the Avatar menu (logged-in only) is opened. We're using this metric to gauge conversion of the "Share Monitor" link in this menu.